index.js

```diff
@@ -30,6 +38,49 @@ app.get('/ideas', (req, res) => {
     res.status(200).json(getIdeas());
 });
 
+app.get('/ideas/sorted/donationSum', (req, res) => {
```

Comment:
You can have a simplified route, for example: `/ideas`

And then you can pass query parameters to modify query, for example: `/ideas?sortBy=donationSum&sortOrder=asc` - to get all ideas sorted by donation sum in ascending order.

Based on this example, try to rethink and redesign all other endpoints.

Reply:
FINALLY! Updated and tested. THANK YOU very much for the knowledge and experience !